jekyll/_cci2/set-up-multiple-configuration-files-for-a-project.adoc

```diff
@@ -29,23 +29,25 @@ In the GitHub repository for your project add a new CircleCI configuration file
 
 As your repository is set up as a project in CircleCI, a `config.yml` file will exist already. If you do not see `.circleci/config.yml`, check which branch you are on. You may have set this up on a non-default branch during the set up process.
```

Comment:
In the new world, a `config.yml` is no longer strictly necessary actually, so I think we want to change this

Reply:
And in the sentence above, on which I can't comment directly, we're forcing users to create the new YML files inside the `.circelci/` directory of the same repo. In reality now you can define pipelines that have config anywhere (e.g. not only in that subdirectory + not necessarily in that same repo). 

I think we might need to flip all this and make it "pipeline centric" rather than "config centric", if you know what I mean?